app/src/main/java/com/pajato/android/gamechat/exp/Checkerboard.java

```diff
@@ -45,14 +45,14 @@
 
     // Private class constants.
 
-    /** The amount of space taken by a smart phone toolbar in dp units. */
-    private static final int SMART_PHONE_HEIGHT = 56;
+    /** The space taken by a smart phone notification bar and digital buttons in dp units. */
+    private static final int SMART_PHONE_HEIGHT = 84; // 56 * 1.5
 
-    /** The amount of space taken by a tablet toolbar in dp units. */
-    private static final int TABLET_HEIGHT = 64;
+    /** The space taken by a tablet notification bar and digital buttons in dp units. */
+    private static final int TABLET_HEIGHT = 42; // 28 * 1.5
 
     /** The amount of vertical space allocated for the player controls. */
-    private static final int CONTROLS_HEIGHT = 112;
+    private static int CONTROLS_HEIGHT;
```

Comment:
If this is no longer a constant, the case should be changed, and the declaration might be instance vs class.  I'll comment further when I see how it is used.